package.json

```diff
@@ -12,13 +12,19 @@
     "ts-node": "^10.5.0"
   },
   "devDependencies": {
-    "typescript": "^4.5.5"
+    "typescript": "^4.5.5",
+    "@typescript-eslint/eslint-plugin": "^5.4.0",
+    "@typescript-eslint/parser": "^5.4.0",
+    "eslint": "7.11.0",
+    "eslint-plugin-import": "~2.25.4"
   },
   "scripts": {
     "build": "tsc",
-    "start": "node ./build/app.js",
-    "dev": "ts-node ./src/app.ts",
-    "clean": "rm -r build"
+    "start": "node ./build/main.js",
+    "dev": "ts-node ./src/main.ts",
+    "clean": "rm -r build",
+    "lint": "eslint './src/**/*.ts' --cache",
+    "lint:fix": "yarn lint --fix"
   },
   "publishConfig": {
     "@mj426382:registry": "https://npm.pkg.github.com"
```

Comment:
is this needed?

Reply:
dev is useful, we can remove clean

Reply:
what about publish config? I think that's only for publishing NPM packages

Reply:
For future hah

Reply:
Deleted